neps/nep-0245.md

```diff
@@ -92,21 +92,17 @@ type Token = {
 // * `token_id`: the token to transfer
 // * `amount`: the number of tokens to transfer, wrapped in quotes and treated
 //    like a string, although the number will be stored as an unsigned integer
-//    with 128 bits.
-// * `approval` (optional): is a tuple of [`owner_id`,`approval_id`]. 
-//   `owner_id` is the valid Near account that owns the tokens.   
-//   `approval_id` is the expected approval ID. A number smaller than
+//    with 128 bits. This value is ignored if the token is an NFT.
+// * `approval_id` (optional): the expected approval ID. A number smaller than
 //    2^53, and therefore representable as JSON. See Approval Management
 //    standard for full explanation. 
 // * `memo` (optional): for use cases that may benefit from indexing or
 //    providing information for a transfer
-
-
 function mt_transfer(
   receiver_id: string,
   token_id: string,
-  amount: string,
-  approval: [owner_id: string, approval_id: number]|null,
+  amount: string, 
+  approval_id: number|null,
```

Comment:
Same goes for approvals, do we really want this to be a part of the interface.
Why don't we just do the barebones MT standard without the extensions and then figure out how those will play together with what we have?